### PR TITLE
[NOID] Bump Arrow versions based on Neo netty bump

### PR DESCRIPTION
**Pull request by gem-neo4j**

### CODE REVIEW

**Review by thelonelyvulpes — APPROVED** [review]
Many thanks

File: LICENSES.txt [hunk 1]
```diff
@@ -3059,6 +3068,7 @@ MIT
   reactive-streams-1.0.4.jar
   slf4j-api-1.7.36.jar
   slf4j-api-2.0.11.jar
+  slf4j-api-2.0.9.jar
```
Comment (anderson4j) [inline LICENSES.txt:3071]: All other versions are bumped, here we include an older instead. What gives?

File: NOTICE.txt [hunk 2]
```diff
@@ -459,6 +468,7 @@ MIT
   reactive-streams-1.0.4.jar
   slf4j-api-1.7.36.jar
   slf4j-api-2.0.11.jar
+  slf4j-api-2.0.9.jar
```
Comment (anderson4j) [inline NOTICE.txt:471]: Same as the other comment, so you see both in case it's a mistake

File: LICENSES.txt (hunk 1 above)
Comment (gem-neo4j) [reply LICENSES.txt:3071]: This isn't a mistake, these files are auto-generated, it just means that arrow-vector is transitively dependent on 2.0.9

**Review by anderson4j — APPROVED** [review]
Good stuff

**Pull request merged**